lms/djangoapps/grades/models.py

```diff
@@ -529,7 +529,7 @@ class Meta(object):
     course_id = CourseKeyField(blank=False, max_length=255)
 
     # Information relating to the state of content when grade was calculated
-    course_edited_timestamp = models.DateTimeField(u'Last content edit timestamp', blank=False)
+    course_edited_timestamp = models.DateTimeField(u'Last content edit timestamp', blank=True, null=True)
```

Comment:
So before, would trying to create a course grade for a course with no recorded edit timestamp have caused the same error we saw with subsections, and we just didn't get to that point?